### PR TITLE
Add frame for winner of veldedighetsfest auction object "Ramme"

### PR DESCRIPTION
**Pull request by danielyanghansen**
# Description

Add frame for winner of veldedighetsfest auction object "Ramme".
(It was quite cheap actually :money_mouth_face: )

# Result

If you've made visual changes, please check the boxes below and include images showing the changes. Descriptions are appreciated.

- [X] Changes look good on both light and dark theme.
- [X] Changes look good with different viewports (mobile, tablet, etc.).
- [X] Changes look good with slower Internet connections.

> [!CAUTION]
> Make sure your images do not contain any real user information.

<table>
    <tr>
        <td width="25%">Description</td>
        <td>Before</td>
        <td>After</td>
    </tr>
    <tr>
        <td>
            Frame for user 7325
        </td>
        <td>
            It wasn't there
        </td>
        <td>
            It is there now
        </td>
    </tr>
</table>

# Testing

No testing needed. I have verified that the user ID (7325) is correct for the winner of the auction
---

Resolves ... (either GitHub issue or Linear task)

### CODE REVIEW

File: app/routes/users/components/UserProfile/index.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ const UserProfile = () => {
   }
 
   //If you wonder what this is, ask somebody
-  const FRAMEID = [6050, 5962, 7276, 7434, 7747, 8493];
+  const FRAMEID = [6050, 5962, 7276, 7325, 7434, 7747, 8493];
```
Comment (itsisak) [inline app/routes/users/components/UserProfile/index.tsx:126]: 🥺
```suggestion
  const FRAMEID = [6050, 5962, 7276,  8476, 7434, 7747, 8493];
```

**Review by itsisak — APPROVED** [review]

File: app/routes/users/components/UserProfile/index.tsx (hunk 1 above)
Comment (danielyanghansen) [reply app/routes/users/components/UserProfile/index.tsx:126]: image

Comment (itsisak) [reply app/routes/users/components/UserProfile/index.tsx:126]: NOK 470, take it or leave it

Comment (danielyanghansen) [reply app/routes/users/components/UserProfile/index.tsx:126]: :no_entry_sign:

Comment (itsisak) [reply app/routes/users/components/UserProfile/index.tsx:126]: 😢

**Review by falbru — APPROVED** [review]

**Review by ShaileshS1702 — APPROVED** [review]

**Review by eikhr — APPROVED** [review]

**Review by EliHaugu — APPROVED** [review]

**Review by sondrejk — APPROVED** [review]
Lgtm!

**Review by Viljen789 — APPROVED** [review]
LGTM

**Review by ch0rizo — COMMENTED** [review]
LGMT!

**Pull request merged**